docs/components/modeler/desktop-modeler/lint-rules/lint-rules.md

```diff
@@ -0,0 +1,82 @@
+---
+id: lint-rules
+title: Lint rules
+description: "Learn how to contribute additional validation (lint) rules to the Desktop Modeler through lint rule plug-ins."
+---
+
+The Desktop Modeler uses [bpmnlint](https://github.com/bpmn-io/bpmnlint) to validate BPMN diagrams. Through defined modeler extension points you add plug-in additional lint rules or configure/disable existing rules.
+
+## Getting started
+
+To get started, you can start from the [custom-linter-rules-plugin template](https://github.com/camunda/camunda-modeler-custom-linter-rules-plugin).
+
+Take the following steps:
+
+1. Clone or fork the repository:
+
+```
+git clone https://github.com/camunda/camunda-modeler-custom-linter-rules-plugin.git
+```
+
+The plugin starter project comes with a client folder which is referenced in the [plugin entry point](../plugins#plugin-entry-point). It contains the script that adds a bpmnlint plugin to the modeler. Since this is a [client plugin](../plugins#extend-the-modeler-and-its-bpmn-and-dmn-components) you need to bundle it.
+
+2. Install the dependencies:
+
+```
+npm install
+```
+
+3. Add a custom rule
+
+To add a custom rule, add it to the [`bpmnlint-plugin-custom/rules`](https://github.com/camunda/camunda-modeler-custom-linter-rules-plugin/tree/master/bpmnlint-plugin-custom/rules) folder. The example project contains a [`no-manual-task.js`](https://github.com/camunda/camunda-modeler-custom-linter-rules-plugin/blob/master/bpmnlint-plugin-custom/rules/no-manual-task.js) file implements a custom rule.
```

Comment:
Typo: "... file THAT implements a custom rule"

Reply:
Noticed this as well -- adjusted in my latest commit.